### PR TITLE
no.png

### PR DESCRIPTION
**Pull request by yoichinak**
I'm trying to unify the X mark.

### CODE REVIEW

**Comment by PRMerger19** [comment]
@yoichinak : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by jamesmontemagno** [comment]
Good one!

**Comment by jamesmontemagno** [comment]
#sign-off

File: docs/essentials/permissions.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Icon Guide:
 | StorageRead | ![Android supported](~/media/shared/yes.png "Android supported") | ![iOS not supported](~/media/shared/no.png "iOS not supported") | ![UWP not supported](~/media/shared/no.png "UWP not supported") | ![watchOS not supported](~/media/shared/no.png "watchOS not supported") | ![tvOS not supported](~/media/shared/no.png "tvOS not supported") | ![Tizen not supported](~/media/shared/no.png "Tizen not supported") |
 | StorageWrite | ![Android supported](~/media/shared/yes.png "Android supported") | ![iOS not supported](~/media/shared/no.png "iOS not supported") | ![UWP not supported](~/media/shared/no.png "UWP not supported") | ![watchOS not supported](~/media/shared/no.png "watchOS not supported") | ![tvOS not supported](~/media/shared/no.png "tvOS not supported") | ![Tizen not supported](~/media/shared/no.png "Tizen not supported") |
 
-If a permission is marked with a ❌ it will always return `Granted` when checked or requested.
+If a permission is marked with a ![❌](~/media/shared/no.png "❌") it will always return `Granted` when checked or requested.
```
Comment (conceptdev) [inline docs/essentials/permissions.md]: an emoji is not appropriate for alt or title text on an image. these values are used for accessibility (screen readers, etc).

**Review by conceptdev — CHANGES_REQUESTED** [review]
please use accessible alt/title values on images

File: docs/essentials/permissions.md (hunk 1 above)
Comment (yoichinak) [reply docs/essentials/permissions.md]: Thank you for your advice 😄

**Review by conceptdev — APPROVED** [review]
Thanks for taking the time to improve the docs, and in this case improve accessibility.

**Comment by conceptdev** [comment]
#sign-off

**Pull request merged**